openapi.yaml

```diff
@@ -4602,6 +4650,7 @@ paths:
     get:
       tags:
         - Activities
+      x-sdk-operation: listWorkspace
```

Comment:
Activities usually live at the top level, outside of a workspace. This endpoint lists the activities under a given workspace, so I've set this operation name to `listWorkspace` to reflect that.

In the Node SDK, this will be access via `whispir.activities.listWorkspace({ workspaceId: 'myId' });`, open to other naming suggestions here.